examples/policies/l3/egress-allow-all/egress-allow-all.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: "allow-all-from-victim"
```

Comment:
What's the thought behind calling it "from victim" here?

Reply:
I didn't really know what else to call it, I figured we'd keep it this way as a mirror image of ingress allow-all. I could also spin that it's also slightly uplifting to see that the victim can fight back from its attackers :P

Reply:
;) While I appreciate the sentiment, I doubt that people looking at the policy with no context will understand it. It's not strongly important, but I suggest we either come up with witty names that make sense without context, or just name them generically.

Reply:
Changed it to frontend.